assets/locales/en/main.json

```diff
@@ -64,39 +64,42 @@
     "slow-connection-2": "You might want to reload the page and try again. Please note you might lose the latest change to the street. Sorry!",
     "no-connection": "Streetmix is having trouble connecting to the Internet."
   },
-	"errors": {
-		"buttons": {
-			"home-button": "Go to the homepage",
-			"sign-in-button": "Sign in again",
-			"reload-button": "Reload the page",
-			"try-again-button": "Try again",
-			"general-help-buttons": "<a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>. <br /> <button id = 'error-home'onClick = {goHome} > Go to the homepage < /button>"
+	"error": {
+		"button": {
+			"home": "Go to the homepage",
+			"sign-in": "Sign in again",
+			"reload": "Reload the page",
+			"try-again": "Try again",
+      "view-example": "View an example street"
```

Comment:
Extra space indent here. It doesn't affect the validity of the JSON file, but I am a stickler for detail.

Reply:
Actually, I think I know what's going on here. What are you using to edit this? It's using tabs instead of spaces, and we should be consistent and use only spaces for indents.